sql/telemetry_derived/clients_daily_keyed_scalar_aggregates_v1/query.sql

```diff
@@ -99,7 +99,7 @@ aggregated AS (
         MIN(value) AS min,
         AVG(value) AS avg,
         SUM(value) AS sum,
-        COUNT(*) AS count
+        IF(MIN(value) IS NULL, NULL, COUNT(*)) AS count
```

Comment:
Here is a query that, when run, shows an example of why I did this.
```
select
  *
from telemetry_derived.client_probe_counts_v1
where os is null
and app_version = 70
and app_build_id = '20190714093936'
and channel = 'nightly'
and metric = 'browser_errors_collected_count'
limit 100
```

Right now, if a count is visible, an average is computed. But the average is just always 0 since there is no sum. This is misleading since an average might *actually* be 0 due to a sum being 0. In any case, this can be resolved separately by not computing average if there is no sum so it's not the main problem.

Additionally, there is the semantic meaning of `count`. I see it as meaning the number of submissions of a probe (with a value) by a user. The main ping has lots of unset probes in a submission that seem to me like they shouldn't be counted. Not to mention, there is nothing the frontend of glam would need to do with this data of count alone.

So overall it just seemed like it added unnecessary computations (and rows of data) for not much gain.

Reply:
Okay, your explanation makes sense. Is `MIN(value)` the proper value to use to check that the submissions were indeed `null`, and the users shouldn't be counted?

Reply:
I agree `MIN(value)` doesn't seem intuitive here. I tried to look for `EVERY()` so we could do `EVERY(value IS NULL)` but that function doesn't seem to be implemented in BigQuery unfortunately...

I do think `MIN(value)` does what we want here though (https://cloud.google.com/bigquery/docs/reference/standard-sql/functions-and-operators#min):

"Returns NULL if there are zero input rows or expression evaluates to NULL for all rows"

Reply:
Agreed, that does seem to do what we need. Can you add a comment?